pkg/scheduler/framework/interface.go

```diff
@@ -295,6 +295,55 @@ type Plugin interface {
 	Name() string
 }
 
+// RequeuePlugin is an interface that must be implemented by "Requeue" plugins.
+//
+// Note: an Requeue plugin is expected to be lightweight and efficient, so it's not expected to
+// involve expensive calls like accessing external endpoints; otherwise it'd block other
+// Pods' enqueuing in event handlers.
+type RequeuePlugin interface {
+	Plugin
+	// Requeue returns status meaning whether the event can make a Pod, which is rejected by this plugin in the past scheduling cycle, schedulable or not.
+	// It's called before a Pod gets moved from unschedulableQ to backoffQ or activeQ.
+	//
+	// - `pod`: the Pod to be enqueued, which is rejected by this plugin in the past.
+	// - `event`: By which event the pod will be moved back to schedQ/backoffQ.
+	//   - It can be nil when Pod is moved by priodically flush or activate.
```

Comment:
what do you mean by activate?

Should we care about flush? Will the plugins have any information to make a decision in that case?

Reply:
> what do you mean by activate?

I meant activation via `PodsToActivate`. But, I found that it's incorrect because we don't call Requeue for Pods going to activeQ/backoffQ via `PodsToActivate`. I'll just remove this comment.

> Should we care about flush? Will the plugins have any information to make a decision in that case?

Sorry No, that comment is based on the old impl. I've already changed Requeue not to be called by flush.